sphinxcontrib/inmanta/api.py

```diff
@@ -476,21 +644,149 @@ def run(self, module_repo: Optional[str], module_name: str, extra_modules: Seque
     ),
 )
 @click.option("--module", help="The module to generate api docs for", required=True)
-@click.option("--extra-modules", "-m", help="Extra modules that should be loaded to render the docs", multiple=True)
+@click.option(
+    "--extra-modules",
+    "-m",
+    help="Extra modules that should be loaded to render the docs",
+    multiple=True,
+)
 @click.option("--source-repo", help="The repo where the upstream source is located.")
 @click.option("--file", "-f", help="Save the generated result here.", required=True)
-def generate_api_doc(module_repo: Optional[str], module: str, extra_modules: Sequence[str], source_repo: str, file: str):
+def generate_api_doc(
+    module_repo: Optional[str],
+    module: str,
+    extra_modules: Sequence[str],
+    source_repo: str,
+    file: str,
+):
+    """
+    Generate API documentation for module and write it to the provided file.
+    """
+
+    write_auto_doc(extra_modules, module, module_repo, file)
+
+
+@cli.command()
+@click.option(
+    "--module_repo",
+    help=(
+        "The repo where all v1 modules are stored (local file). Ignored for v2 modules, which are always fetched from the"
+        " Python environment."
+    ),
+)
+@click.option(
+    "--module-name", help="The module to generate api docs for", required=True
+)
+@click.option(
+    "--extra-modules",
+    "-m",
+    help="Extra modules that should be loaded to render the docs",
+    multiple=True,
+)
+@click.option(
+    "--out-dir", "-d", help="Place doc output in this directory.", required=True
+)
+def generate_api_doc_v2(
```

Comment:
We should not call this "api doc" because it's a lot more than just the API. For example this might also include the changelog, etc.